CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.1.0 (2020-06-16)
+
+- Added input box support to the datepicker component. [#166](https://github.com/blackbaud/skyux-datetime/pull/166)
```

Comment:
Probably doesn't make much difference, but I wonder if this might read a little clearer at a glance if we fleshed it out to `Added support for the input box component to the datepicker component.` or `Updated the datepicker component to support the input box component.` ... No big deal if you don't think either of those improve this. Just throwing it out there as a possibility.